examples/agent-ports/service-agent.hspec

```diff
@@ -0,0 +1,16 @@
+name: service-agent
+
+image:
+  registry: registry.hub.docker.com
+  name: library/redis
+
+ports:
+  - port: 5050
+
+agents:
+  - name: helloappsidecar
+    image: gcr.io/google-samples/hello-app:1.0
```

Comment:
better to have prom-exporter agent instead of hello-app. we may use https://github.com/mateobur/prometheus-monitoring-guide/blob/master/redis_prometheus_exporter.yaml